pupil-api/src/lib/validator/pin-validator.js

```diff
@@ -0,0 +1,16 @@
+const moment = require('moment')
+
+const pinValidator = {}
+
+/**
+ * Validate pin
+ * @param {string} pin
+ * @param {Moment} pinExpiresAt
+ * @returns {Boolean}
+ */
+pinValidator.isActivePin = (pin, pinExpiresAt) => {
+  if (!pinExpiresAt || !pin) return false
+  return moment(pinExpiresAt).isAfter(moment.utc())
+}
+
+module.exports = pinValidator
```

Comment:
Here, and in the other places where `module.exports` is used, I believe "the typescript way" is to do an `export = whatever_was_exported_before` instead of the `module.exports =` ?

Reply:
yes, however this is still a js file.  i was planning to allow a mix and we convert them gradually.